src/mongo/MongoScrapbook.ts

```diff
@@ -221,7 +218,7 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 			name: '',
 			arguments: [],
 			argumentObjects: [],
-			chained: funcCallCount > 1 ? true : false
+			sendToShell: funcCallCount > 1 ? true : false
```

Comment:
I don't think it's accurate to rename `chained` to `sendToShell`. We use the shell as a catch-all for several different unsupported cases, one of which happens to be `chained`. I would prefer to leave the name as-is and wait for https://github.com/microsoft/vscode-cosmosdb/pull/1105 to do any refactoring related to this

Reply:
This is related to the fix for #1096. When both bugs are fixed (chained commands and the Date), having two separate flags for the same action of sending to the shell is redundant when on flag would do.

Reply:
see other comment near executeCommand